src/test/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/uiTest/SettingsComponentTest.kt

```diff
@@ -0,0 +1,284 @@
+        assertThat(settingsFrame.configurationIdField.isShowing).isTrue
+        assertThat(settingsFrame.executeTestsCheckbox.isShowing).isTrue
+        assertThat(settingsFrame.createAssertionsCheckBox.isShowing).isTrue
+        assertThat(settingsFrame.debugModeCheckbox.isShowing).isTrue
+        assertThat(settingsFrame.minimiseTestSuiteCheckBox.isShowing).isTrue
+        assertThat(settingsFrame.flakyTestCheckBox.isShowing).isTrue
+
+        assertThat(settingsFrame.coverageSeparator.isShowing).isTrue
+        assertThat(settingsFrame.lineCoverageCheckBox.isShowing).isTrue
+        assertThat(settingsFrame.branchCoverageCheckBox.isShowing).isTrue
+        assertThat(settingsFrame.exceptionCoverageCheckBox.isShowing).isTrue
+        assertThat(settingsFrame.mutationCoverageCheckBox.isShowing).isTrue
+        assertThat(settingsFrame.outputCoverageCheckBox.isShowing).isTrue
+        assertThat(settingsFrame.methodCoverageCheckBox.isShowing).isTrue
+        assertThat(settingsFrame.methodNoExcCoverageCheckBox.isShowing).isTrue
+        assertThat(settingsFrame.cBranchCoverageCheckBox.isShowing).isTrue
+    }
+
+    @Order(6)
+    @Test
+    fun changeEvoSuiteTabCoverageSectionValues(remoteRobot: RemoteRobot): Unit = with(remoteRobot) {
+        // Get previous values
+        var settingsFrame = find(SettingsFrame::class.java, timeout = Duration.ofSeconds(15))
+        val prevLineCoverageCheckBoxValue = settingsFrame.lineCoverageCheckBox.isSelected()
+        val prevBranchCoverageCheckBoxValue = settingsFrame.branchCoverageCheckBox.isSelected()
+        val prevExceptionCoverageCheckBoxValue = settingsFrame.exceptionCoverageCheckBox.isSelected()
+        val prevMutationCoverageCheckBoxValue = settingsFrame.mutationCoverageCheckBox.isSelected()
+        val prevOutputCoverageCheckBoxValue = settingsFrame.outputCoverageCheckBox.isSelected()
+        val prevMethodCoverageCheckBoxValue = settingsFrame.methodCoverageCheckBox.isSelected()
+        val prevMethodNoExcCoverageCheckBoxValue = settingsFrame.methodNoExcCoverageCheckBox.isSelected()
+        val prevCBranchCoverageCheckBoxValue = settingsFrame.cBranchCoverageCheckBox.isSelected()
+
+        // Change checkbox values and apply the settings
+        settingsFrame.lineCoverageCheckBox.setValue(!prevLineCoverageCheckBoxValue)
+        settingsFrame.branchCoverageCheckBox.setValue(!prevBranchCoverageCheckBoxValue)
+        settingsFrame.exceptionCoverageCheckBox.setValue(!prevExceptionCoverageCheckBoxValue)
+        settingsFrame.mutationCoverageCheckBox.setValue(!prevMutationCoverageCheckBoxValue)
+        settingsFrame.outputCoverageCheckBox.setValue(!prevOutputCoverageCheckBoxValue)
+        settingsFrame.methodCoverageCheckBox.setValue(!prevMethodCoverageCheckBoxValue)
+        settingsFrame.methodNoExcCoverageCheckBox.setValue(!prevMethodNoExcCoverageCheckBoxValue)
+        settingsFrame.cBranchCoverageCheckBox.setValue(!prevCBranchCoverageCheckBoxValue)
```

Comment:
Is it possible to extract those checkboxes to a list and set them that way, so that there is not so much code duplication in multiple places?

Reply:
Since some of the tests are different in nature BeforeEach and AfterEach cannot be applied. I will look into using a list or helper methods to fix this.